Identity/Identity.API/Program.cs

```diff
@@ -13,17 +18,32 @@
     app.UseSwaggerUI();
 }
 
-app.MapGet("/users", () =>
+app.MapGet("/users", async (IdentityDatabaseContext dbContext) =>
 {
-    var users = new User[]
+    return dbContext.Users.Select(u => new
     {
-        new()
-        {
-            Username = "johndoe"
-        }
+        u.Id,
+        u.Username,
+        u.PasswordHash,
+        u.Email,
+        u.Roles
+    });
```

Comment:
this shows warning for missing await operator. You can use .ToListAsync() here

Reply:
Added .ToListAsync()